clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -125,56 +119,52 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
     ]
   );
 
-  if (!experience.experience_config) {
-    debugLog(options.debug, "No experience config found");
-    return null;
-  }
-  const experienceConfig = experience.experience_config;
-
-  const dispatchOpenBannerEvent = () => {
+  const dispatchOpenBannerEvent = useCallback(() => {
```

Comment:
see https://github.com/ethyca/fides/pull/4356/files#r1376843489 for why we wrap it in a `useCallback`. we avoided the infinite renders before by not including `onOpen` in the dependency array, but that's not a good practice